modin/core/storage_formats/pandas/query_compiler.py

```diff
@@ -147,7 +147,10 @@ def _dt_prop_map(property_name):
 
     def dt_op_builder(df, *args, **kwargs):
         """Access specified date-time property of the passed frame."""
-        prop_val = getattr(df.squeeze(axis=1).dt, property_name)
+        squeezed_df = df.squeeze(axis=1)
+        if isinstance(squeezed_df, pandas.DataFrame) and len(squeezed_df.columns) == 0:
```

Comment:
`squeezed_df` can become an empty Series if `df` is originally a DataFrame with a single column but no rows.

If `squeezed_df` becomes an empty Series, there are 2 possible outcomes. If the Series dtype is `datetime64[ns]`, it should return an empty DataFrame as expected. If the Series dtype is anything else, it will error because `dt` is only accessible to datetimelike values.

Is there a situation where the passed in `df` would be a DataFrame with a single column but no rows and the dtype is not `datetime64[ns]` when it is supposed to be?

Reply:
I can't think of such a case.